collect_app/src/main/java/org/odk/collect/android/widgets/viewmodels/DateTimeViewModel.java

```diff
@@ -0,0 +1,54 @@
+package org.odk.collect.android.widgets.viewmodels;
+
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDateTime;
+import org.odk.collect.android.utilities.DateTimeUtils;
+
+public class DateTimeViewModel extends ViewModel {
+    private final MutableLiveData<LocalDateTime> selectedDate = new MutableLiveData<>();
+    private final MutableLiveData<DateTime> selectedTime = new MutableLiveData<>();
+
+    private final DatePickerDialog.OnDateSetListener dateSetListener = (view, year, monthOfYear, dayOfMonth) -> {
+        view.clearFocus();
```

Comment:
Do we need this? I know it was needed in [TimePickerDialog](https://github.com/getodk/collect/issues/3948) but here?

Reply:
Oh okay I'll check once

Reply:
@grzesiek2010 I found that `view.clearFocus()` does not make any difference on the DatePicker or TimePicker dialogs. I too remember it was required for the Time Picker dialog to hide the extra window behind the dialog, but even without it, I don't see that window now.

Reply:
In `TimePicker` it is required now as you removed it I can reproduce https://github.com/getodk/collect/issues/3948 again.